FAQ.md

```diff
@@ -16,16 +25,32 @@ Some possible issues:
  * Make sure your system time is (semi-)correct (let it autosync with an online server)
  * Make sure the mobile app is not on (kill the process if need be)
  * Make sure the account you're botting on did do the initial tutorial (mainly catching a starter Pokemon)
+ * Make sure your item bank is empty
+ * You catched too many pokemons in a short time and are softbanned (don't catch more than 1000 Pokemon in 23 hours, [source](https://www.reddit.com/r/pokemongodev/comments/4xkqmq/new_ban_types_and_their_causes/)).
+
+## The bot doesn't find/loot Pokestops
+
+Some possible issues:
+
+ * Your startlocation is wrong and/or there are no pokestops nearby.
+ * You looted too many pokestops in a short time and are softbanned (don't loot more than 2000 Pokestops in 23 hours, [source](https://www.reddit.com/r/pokemongodev/comments/4xkqmq/new_ban_types_and_their_causes/))
+
+## The bot doesn't walk
+
+Some possible issues:
+
+ * You're camping a Pokestop with a lure (should be visible in log)
+ * You're resting (should be visible in log)
 
 ## Immediately after starting I get a LoginFailedException
 
 - Make sure the provided credentials in the `config.properties` file are correct.
 - If you're using PTC, your credentials are correct and your password is longer than 15 characters, only enter the first 15 characters of your account and the login should work.
 - If there is a token present, remove it and retry.
 
-## After 1.5 hours I get a errors with Google login
+## After 1.5 hours I get a errors with PTC/Google login `LoginFailedException: Invalid Auth status code recieved, token not refreshed?`
 
-Known issue with the Java API handling login for Google.
+Known issue with the Java API handling login for PTC/Google.
```

Comment:
With Google I had it exactly every 1.5h, with PTC it can vary but is mostly around 1.5h-ish.

The token field is also never filled in anymore in the config.properties, maybe it has to do something with that?